src/category_theory/comma.lean

```diff
@@ -351,4 +457,59 @@ end
 
 end under
 
+section
+variables (T)
+
+/-- The arrow category of `T` has as objects all morphisms in `T` and as morphisms commutative
+     squares in `T`. -/
+@[derive category]
+def arrow := comma.{v₃ v₃ v₃} (𝟭 T) (𝟭 T)
```

Comment:
Can you save restating a bunch of properties and simp-lemmas by making this an `abbreviation`?

Reply:
I tried this, but it seemed not to make any difference.